jenkins/jenkins-on-azure/index.html

```diff
@@ -0,0 +1,46 @@
+<html>
+
+<head>
+  <link rel="stylesheet" type="text/css" href="/jenkins-on-azure/site.css">
+  <title>Jenkins On Azure</title>
+</head>
+
+<body>
+  <div id="page-head">
+    <div id="header">
+      <div class="logo">
+        <a id="jenkins-home-link" href="/ ">
+          <img id="jenkins-head-icon" alt="title " src="/jenkins-on-azure/headshot.png ">
+          <img id="jenkins-name-icon" height="34" alt="title " width="139 " src="/jenkins-on-azure/title.png ">
+        </a>
+      </div>
+    </div>
+  </div>
+  <div id="breadcrumbBar">
+    <div style="height: 28px;"></div>
+    <div class="top-sticker noedge">
+      <ul id="breadcrumbs">
+        <li class="item"><a class="model-link inside breadcrumbBarAnchor" href="/">Jenkins</a></li>
+      </ul>
+    </div>
+  </div>
+  </div>
+
+  <div id="feature">
+    <div id="image">
+      <img src="/jenkins-on-azure/azure.svg" />
+    </div>
+    <div id="content">
+      <h1>Jenkins On Azure</h1>
+      <div id="content_body">
+        <p>You need to connect to the Jenkins instance using SSH port forwarding!</p>
```

Comment:
I would tell them _why_ they need to ssh. Something like "This Jenkins instance does not support https, so logging in through a public IP address has been disabled (it would expose your password and other information to eavesdropping). To securely login, you need to connect to the Jenkins instance using SSH port forwarding"